### ISSUE TITLE
Warning with tile

### ISSUE THREAD

**Issue by gen4sp** [shoutem/ui#300]
No any style applied to tile tag. 
just taken from example
```
<TouchableOpacity key={index} onPress={this.openActionModal.bind(this, rowData[0].id)}>
          <Image
            styleName="large"
            source={{ uri: rowData[0].image.url }}
          >
            <Tile> //WARNIN HERE
              <Title styleName="md-gutter-bottom">{rowData[0].name}</Title>
              <Subtitle styleName="sm-gutter-horizontal">{rowData[0].address}</Subtitle>
            </Tile>
          </Image>
          <Divider styleName="line" />
        </TouchableOpacity>
```

the warning:
```
Warning: Failed prop type: Invalid props.style key `heroAnimation` supplied to `AnimatedComponent`.
Bad object: {
  "flex": 1,
  "flexDirection": "column",
  "alignItems": "center",
  "backgroundColor": "rgba(0, 0, 0, 0.2)",
  "alignSelf": "stretch",
  "justifyContent": "center",
  "paddingLeft": 25,
  "paddingRight": 25,
  "paddingTop": 45,
  "paddingBottom": 40,
  "position": "absolute",
  "top": 0,
  "left": 0,
  "right": 0,
  "bottom": 0
}
Valid keys: [
  "display",
  "width",
  "height",
  "top",
  "left",
  "right",
  "bottom",
  "minWidth",
  "maxWidth",
  "minHeight",
  "maxHeight",
  "margin",
  "marginVertical",
  "marginHorizontal",
  "marginTop",
  "marginBottom",
  "marginLeft",
  "marginRight",
  "padding",
  "paddingVertical",
  "paddingHorizontal",
  "paddingTop",
  "paddingBottom",
  "paddingLeft",
  "paddingRight",
  "borderWidth",
  "borderTopWidth",
  "borderRightWidth",
  "borderBottomWidth",
  "borderLeftWidth",
  "position",
  "flexDirection",
  "flexWrap",
  "justifyContent",
  "alignItems",
  "alignSelf",
  "alignContent",
  "overflow",
  "flex",
  "flexGrow",
  "flexShrink",
  "flexBasis",
  "aspectRatio",
  "zIndex",
  "direction",
  "shadowColor",
  "shadowOffset",
  "shadowOpacity",
  "shadowRadius",
  "transform",
  "transformMatrix",
  "decomposedMatrix",
  "scaleX",
  "scaleY",
  "rotation",
  "translateX",
  "translateY",
  "backfaceVisibility",
  "backgroundColor",
  "borderColor",
  "borderTopColor",
  "borderRightColor",
  "borderBottomColor",
  "borderLeftColor",
  "borderRadius",
  "borderTopLeftRadius",
  "borderTopRightRadius",
  "borderBottomLeftRadius",
  "borderBottomRightRadius",
  "borderStyle",
  "opacity",
  "elevation"
]
    in AnimatedComponent (at connectStyle.js:247)
    in Styled(AnimatedComponent) (at actionsList.js:60)
    in RCTImageView (at Image.ios.js:371)
    in Image (at Image.js:98)
    in Image (at connectAnimation.js:240)
    in AnimatedComponent (at connectStyle.js:247)
    in Styled(AnimatedComponent) (at actionsList.js:56)
    in RCTView (at View.js:132)
    in View (at AnimatedImplementation.js:1844)
    in AnimatedComponent (at TouchableOpacity.js:178)
    in TouchableOpacity (at TouchableOpacity.js:18)
    in TouchableOpacity (at connectStyle.js:247)
    in Styled(TouchableOpacity) (at actionsList.js:55)
    in StaticRenderer (at ListView.js:442)
    in RCTScrollContentView (at ScrollView.js:667)
    in RCTScrollView (at ScrollView.js:756)
    in ScrollView (at ListView.js:321)
    in ListView (at ListView.js:305)
    in ListView (at connectStyle.js:247)
    in Styled(ListView) (at actionsList.js:112)
    in RCTView (at View.js:132)
    in View (at connectAnimation.js:240)
    in AnimatedComponent (at connectStyle.js:247)
    in Styled(AnimatedComponent) (at actionsList.js:111)
    in ActionList (at main.js:62)
    in _ (created by SceneComponent)
    in SceneComponent (at SceneMap.js:13)
    in RCTView (at View.js:132)
    in View (at TabViewPagerScroll.js:139)
    in RCTScrollContentView (at ScrollView.js:667)
    in RCTScrollView (at ScrollView.js:756)
    in ScrollView (at TabViewPagerScroll.js:118)
    in TabViewPagerScroll (at TabViewAnimated.js:84)
    in RCTView (at View.js:132)
    in View (at TabViewAnimated.js:279)
    in TabViewAnimated (at main.js:71)
    in MainScreen (at SceneView.js:35)
    in SceneView (at CardStack.js:393)
    in RCTView (at View.js:132)
    in View (at CardStack.js:392)
    in RCTView (at View.js:132)
    in View (at CardStack.js:391)
    in RCTView (at View.js:132)
    in View (at AnimatedImplementation.js:1844)
    in AnimatedComponent (at Card.js:27)
    in Card (at PointerEventsContainer.js:55)
    in Container (at CardStack.js:430)
    in RCTView (at View.js:132)
    in View (at CardStack.js:364)
    in RCTView (at View.js:132)
    in View (at CardStack.js:363)
    in CardStack (at CardStackTransitioner.js:110)
    in RCTView (at View.js:132)
    in View (at Transitioner.js:187)
    in Transitioner (at CardStackTransitioner.js:60)
    in CardStackTransitioner (at StackNavigator.js:52)
    in Unknown (at createNavigator.js:33)
    in Navigator (at createNavigationContainer.js:202)
    in NavigationContainer (at registerRootComponent.js:24)
    in PureContainer (at ModalHost.js:88)
    in RCTView (at View.js:132)
    in View (at ModalHost.js:87)
    in ModalHost (at registerRootComponent.js:23)
    in ExponentRootComponent (at renderApplication.js:35)
    in RCTView (at View.js:132)
    in View (at AppContainer.js:98)
    in RCTView (at View.js:132)
    in View (at AppContainer.js:97)
    in AppContainer (at renderApplication.js:34)
```

**Comment by vinhtq** [comment]
+1

**Comment by Vladimir-Vdovic** [comment]
Issue resolved with version 0.21.3 of the UI toolkit. Tile components no longer produce warnings:
```
Warning: Failed prop type: Invalid props.style key `heroAnimation` supplied to `AnimatedComponent`.
```

Closing issue.
